sklearn/multioutput.py

```diff
@@ -848,7 +897,7 @@ class ClassifierChain(MetaEstimatorMixin, ClassifierMixin, _BaseChain):
 
     Parameters
     ----------
-    base_estimator : estimator
+    estimator : estimator, default=None
```

Comment:
This is an API change, we should not do that. We should keep estimator a positional argument without defautl argument.